spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/DefaultAccessTokenConverter.java

```diff
@@ -55,14 +55,32 @@ public void setUserTokenConverter(UserAuthenticationConverter userTokenConverter
 	}
 
 	/**
-	 * Flag to indicate the the grant type should be included in the converted token.
+	 * Converter for the part of the data in the token representing a user.
+	 *
+	 * @return the set userTokenConverter
+	 */
+	public UserAuthenticationConverter getUserTokenConverter() {
+		return userTokenConverter;
+	}
+
+	/**
+	 * Flag to indicate that the grant type should be included in the converted token.
 	 * 
 	 * @param includeGrantType the flag value (default false)
 	 */
 	public void setIncludeGrantType(boolean includeGrantType) {
 		this.includeGrantType = includeGrantType;	
 	}
 
+	/**
+	 * Flag to indicate that the grant type should be included in the converted token.
+	 *
+	 * @return the flag value
+	 */
+	public boolean isIncludeGrantType() {
```

Comment:
This does not need to be exposed. It's only required internally for `convertAccessToken` and `extractAuthentication`.